content/blog/2025/logical-shorthand.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Support logical shorthands in CSS
+sub: Can we move this process forward?
+date: 2025-03-20
+author: miriam
+image:
+  src: blog/2025/logical-demo.jpg
+  alt: >
+    An email signup form,
+    once in English (left to right)
+    and then in Arabic (right to left)
+    with the rounded input field and signup button
+    attached at a straight border
+tags:
+  - CSS
+  - CSSWG
+action:
+  text: Sponsor work on logical shorthands
+  url: https://opencollective.com/oddbird-open-source/
+summary: >
+  The CSS Working Group recently resolved
+  to add a `size` shorthand for setting
+  both the `width` and `height` of an element.
+  Many people asked about setting the 'logical'
```

Comment:
I could be reading this wrong but I keep wondering if there are words missing after 'logical'. Should it be "setting the ''logical shorthand properties for `inline-size` and `block-size`"?

Reply:
it should!

Reply:
well kinda, see if that change makes more sense.

Reply:
Much clearer now 👍🏽